public/wave/docs

```diff
@@ -0,0 +1 @@
+/home/andrii/Projects/wave-stripe/wave/docs
```

Comment:
Where we use this file?

Reply:
this is wave feature. it doesn't ignore in git some files to create ready template.  It will be overwritten on the initiation of the app

Reply:
this is simlink, not file